### DISCUSSION TITLE
How to translate HTTP Error Messages?

### DISCUSSION THREAD

**Discussion by Barbapapazes** [laravel/framework#53393] [Q&A]
Hello,

I would like to translate this error message: https://github.com/search?q=repo%3Alaravel%2Fframework+%22Too+Many+Attempts.%22&type=code

But I'm unsable to understand how I should do it.

**Comment by noefleury** [comment]
Hi,

It seems not possible to auto-translate this type of exception (unlike ValidationException).

One way to accomplish what you want should be to customize the exception rendering logic, see : https://laravel.com/docs/11.x/errors#rendering-exceptions
